addon/ng2/blueprints/ng2/files/config/karma.conf.js

```diff
@@ -24,8 +25,19 @@ module.exports = function (config) {
     preprocessors: {
       './src/test.ts': ['angular-cli']
     },
+    coverageReporter: {
+      dir: 'coverage',
+      reporters: [
+        { type : 'json', subdir: '.', file: 'coverage-final.json'}
+      ]
+    },
```

Comment:
Is there a reason to keep using `karma-coverage`? `karma-remap-istanbul` produces the coverage report and no longer needs the intermediate file and instrumenting is done by `sourcemap-istanbul-instrumenter-loader`.